build.gradle

```diff
@@ -74,18 +90,32 @@ task sourcesJar(type: Jar) {
 task groovydocJar(type: Jar) {
     classifier = 'groovydoc'
     from groovydoc
-} 
+}
 
 artifacts {
     archives groovydocJar, sourcesJar
 }
 
 if (!version.endsWith("SNAPSHOT")) {
-    signing {           
+    signing {
         sign configurations.archives
     }
 }
 
+pluginBundle {
+  website = 'https://github.com/WASdev/ci.gradle'
+  vcsUrl = 'https://github.com/WASdev/ci.gradle'
+  description = 'Gradle plugins for managing Liberty profile servers'
+  tags = ['liberty', 'websphere', 'devops']
+
+  plugins {
+    ciGradle {
+      id = 'net.wasdev.wlp.gradle.plugins.Liberty'
+      displayName = 'ci.Gradle'
```

Comment:
Not sure where one would see this name, but wondering if the G should be upper case.  Be consistent.

Reply:
I changed it in both places. For some reason, I thought the G was capitalized in the github page.